.github/workflows/build-node.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x, 15.x, 16.x]
+        node-version: [12.x, 14.x, 15.x, 16.x]
```

Comment:
Any specific reason to remove v10?

Reply:
a lot of tests break, so might drop support for it. As chances are, things will break in v2